sui/src/config/mod.rs

```diff
@@ -105,6 +107,31 @@ fn socket_addr_from_hostport(host: &str, port: u16) -> SocketAddr {
         .unwrap_or_else(|| panic!("Hostname/IP resolution failed for {host}"))
 }
 
+// When validators start they need to find peers in the committee
+// Here we give them some time to retry connecting with exponentially back-off
+// TODO: for testnet, we may need to tune the parameters to allow
+// more asynchronous starting.
+fn socket_addr_from_hostport_retry(host: &str, port: u16) -> SocketAddr {
+    let back_off = ExponentialBackoffBuilder::new()
+        .with_initial_interval(Duration::from_secs(2))
+        .with_multiplier(2.0)
+        .with_max_elapsed_time(Some(Duration::from_secs(30)))
```

Comment:
Is 30 seconds enough?  We should really make this a function parameter, and ideally we can change this using environment vars or some other config method.

Reply:
In my tests, 30 seconds work in the most of the time. I don't see it hurts with a slightly higher number - going to adjust it to 2 minutes for now.

Agreed with config, this is one of the things that we should put into a config in the long run